package.json

```diff
@@ -823,21 +823,22 @@
         }
       },
       {
-        "title": "Checkmarx AI Guided Remediation",
+        "title": "Checkmarx AI Security Champion",
         "id": "ask-kics",
         "order": 2,
         "properties": {
-          "CheckmarxAiGuidedRemediation.OpenAI Key": {
+          "CheckmarxSecurityChampion.key": {
+            "title": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
Was this on purpose or something it was left behind?

Reply:
Left behind. Good catch :)